app/routes/dashboard/index.tsx

```diff
@@ -19,7 +19,7 @@ export const action: ActionFunction = async ({ request }) => {
   const { userId, headers } = await ensureUserCookie(request);
   try {
     const project = await db.project.create({ title, userId });
-    return redirect(`${config.designerPath}/${project.id}`, { headers });
+    return redirect(`${config.designerPath}/${project?.id}`, { headers });
```

Comment:
I think project can't be empty here, at least from looking at the code and types I don't see how

Reply:
`await prisma.project.create(` doesn't return null based on types, not sure why parseProject assumes it can be null, is prisma types wrong there?

Reply:
maybe its the switch from mongodb to PostgreSQL that changed the types?

Reply:
It's because of findUnique and that can return null :/